### PR TITLE
Update gdpr-for-on-premises-file-shares.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the AIP scanner link

### CODE REVIEW

**Comment by msbemba** [comment]
@johanfreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2778

File: compliance/regulatory/gdpr-for-on-premises-file-shares.md [hunk 1]
```diff
@@ -51,6 +51,6 @@ The recommended approach for files shares includes these steps:
 
     -   To find personal data for data subject requests, use Azure Information Protection scanner. You can also configure SharePoint Server search to crawl file shares.
 
-For more information on using Azure Information Protection scanner to find and label personal data, see the Microsoft GDPR Data Discovery Toolkit at [https://aka.ms/gdprpartners](<https://aka.ms/gdprpartners>).
+For more information on using Azure Information Protection scanner to find and label personal data, see [Deploy AIP Scanner](<https://docs.microsoft.com/azure/information-protection/deploy-aip-scanner).
```
Comment (JohanFreelancer9) [inline compliance/regulatory/gdpr-for-on-premises-file-shares.md:54]: Not correctly linked. Should be (https://docs.microsoft.com/azure/information-protection/deploy-aip-scanner).

**Comment by msbemba** [comment]
I have created another PR#https://github.com/MicrosoftDocs/Compliance/pull/7 to fix the incorrect URL

CC: @MikePlumleyMSFT

**Pull request merged**